docs/02-app/01-building-your-application/07-configuring/05-mdx.mdx

```diff
@@ -643,10 +643,14 @@ const withMDX = createMDX({
   },
 })
 
-// Wrap MDX and Next.js config with each other
+// Combine MDX and Next.js config
 export default withMDX(nextConfig)
 ```
 
+> **Good to know**:
+>
+> remark and rehype plugins cannot be used with [Turbopack](/docs/architecture/turbopack), due to lack of plugins support in [`mdxRs`](#using-the-rust-based-mdx-compiler-experimental)
```

Comment:
This message is wrong and the good to know [here](https://nextjs.org/docs/app/building-your-application/configuring/mdx#using-the-rust-based-mdx-compiler-experimental) is also outdated.

- Turbopack works with `@next/mdx`
- The reason you can't use `remark` / `rehype` plugins with Turbopack is that they are JavaScript functions and you can't pass JavaScript to Rust. Can link to this issue: https://github.com/vercel/next.js/issues/71819

Reply:
Oh sorry, wasn't aware of that detail! PR open:

- https://github.com/vercel/next.js/pull/72467